packages/pools/src/routes.ts

```diff
@@ -270,6 +272,14 @@ export class OptimizedRoutes {
       return {
         ...routes[i],
         amount,
+        isMultihopOsmoFeeDiscount: isOsmoRoutedMultihop(
+          routes[0].pools.map((routePool) => ({
```

Comment:
We shouldn't need to modify this component. We can get `multiHopOsmoDiscount` from the router in the trade clipboard, then IMO it makes sense to just pass that flag into the trade route component

Reply:
You are right, thanks! Let me update this